src/MICore/UnixUtilities.cs

```diff
@@ -194,12 +194,12 @@ public static bool IsBinarySigned(string filePath, Logger logger)
 
             p.OutputDataReceived += (sender, e) =>
             {
-                OutputNonEmptyString(e.Data, "stdout: ", logger);
+                OutputNonEmptyString(e.Data, "binSign-stdout: ", logger);
             };
 
             p.ErrorDataReceived += (sender, e) =>
             {
-                OutputNonEmptyString(e.Data, "stderr: ", logger);
+                OutputNonEmptyString(e.Data, "binSign-stderr: ", logger);
```

Comment:
Please either call this "verifySign" or "codeSign". "binSign" seems a little weird.